core/frontend/src/render/GraphicBuilder.ts

```diff
@@ -147,6 +147,12 @@ export interface GraphicBuilderOptions {
    * @note Edges will tend to z-fight with their surfaces unless the graphic is [[pickable]].
    */
   generateEdges?: boolean;
+
+  /** If defined, specifies a point about which the graphic will rotate such that it always faces the viewer.
+   * This can be particular useful for planar regions to create a billboarding effect - e.g., to implement [[Marker]]-like WebGL decorations.
+   * @note This has no effect for graphics displayed in a 2d view.
+   */
+  viewIndependentOrigin?: Point3d;
```

Comment:
Is viewIndependentOrigin a world coordinate point, or is the placementTransform applied to it?

I assume this works with cached decorations (not really needed for uncached)...

Reply:
I think it would be helpful just to add a note that the branch transform is applied the point but the placement transform is not?

Reply:
better?